### ISSUE TITLE
Issue with Spacing on Fonts with Different sizes

### ISSUE THREAD

**Issue by boblete** [tleunen/pixi-multistyle-text#11]
Hi I'm using a combination of New Lines and word wrap - when I do this I get really odd layout. (Pixi 3.0.10) 

```
 this.MULTI_STYLE_TEXT =  {
        default: { font: "14px proxima-nova-alt-regular", fill: "#000000",align: "left",stroke: "#FFFFFF", strokeThickness: 0  },
        h: {font: "20px proxima-nova-alt-bold" },
        h1: {font: "18px proxima-nova-alt-bold"},
        h2: {font: "14px proxima-nova-alt-bold" },
        h3: {font: "16px proxima-nova-alt-bold" },
        normal:{font: "14px proxima-nova-alt-regular" },
        p:{font: "14px proxima-nova-alt-regular" },
        br:{font: "14px proxima-nova-alt-regular" }
       
    };
this.txt = new PIXI.MultiStyleText(cText,this.MULTI_STYLE_TEXT,{align:"left",wordWrap: true, wordWrapWidth:520 } );
```

![screen shot 2016-12-21 at 16 29 54](https://cloud.githubusercontent.com/assets/445490/21397189/c1504d7e-c79a-11e6-962c-a5a15bc57053.png)

**Comment by bluepichu** [comment]
Interesting - I'll try to take a look later today.

**Comment by bluepichu** [comment]
The bug is that each line's Y position seems to assume that all previous lines had the same height that it did.  A fix will be coming shortly.

**Comment by bluepichu** [comment]
See 63b946e for the fixed version.  This fix is dependent on #14, so it can't be merged in until that gets resolved.
